modules/battenberg/1.1/src/battenberg_wgs_hg38.R

```diff
@@ -75,18 +65,29 @@ MIN_BASE_QUAL = 20
 MIN_MAP_QUAL = 35
 CALC_SEG_BAF_OPTION = 3
 
-# WGS specific static
-ALLELECOUNTER = "alleleCounter" #conda package that should have this: cancerit-allelecount
-PROBLEMLOCI = paste0(REFERENCE_BASE,"/probloci_270415.txt.gz")
-
-print(PROBLEMLOCI);
 
 # Change to work directory and load the chromosome information
 original_dir = getwd()
 setwd(RUN_DIR)
 NORMALBAM = paste0(normalizePath(original_dir,"\\"), "/",opt$nb)
 TUMOURBAM = paste0(normalizePath(original_dir,"\\"), "/",opt$tb)
 
+REFERENCE_BASE = paste0(normalizePath(original_dir,"\\"), "/",opt$ref)
+IMPUTEINFOFILE = paste0(REFERENCE_BASE,"/impute_info.txt")
+print(IMPUTEINFOFILE)
+
+REPLICCORRECTPREFIX = paste0(REFERENCE_BASE, "/battenberg_wgs_replic_correction_1000g_v3/1000_genomes_replication_timing_chr_")
+G1000PREFIX = paste0(REFERENCE_BASE, "/battenberg_1000genomesloci2012_v3/1000genomesAlleles2012_chr")
+G1000PREFIX_AC = paste0(REFERENCE_BASE, "/battenberg_1000genomesloci2012_v3/1000genomesloci2012_chr")
+GCCORRECTPREFIX = paste0(REFERENCE_BASE, "/battenberg_wgs_gc_correction_1000g_v3/1000_genomes_GC_corr_chr_")
+IMPUTE_EXE = "impute2"  #install using conda
+
+# WGS specific static
+ALLELECOUNTER = "alleleCounter" #conda package that should have this: cancerit-allelecount
+PROBLEMLOCI = paste0(REFERENCE_BASE, "/probloci.txt.gz")
+
```

Comment:
Can you move these lines back to their original location so it's easier to see what has been changed rather than moved?

Reply:
Done